consensus/ibft/ibft.go

```diff
@@ -604,14 +605,14 @@ func (i *Ibft) buildBlock(snap *Snapshot, parent *types.Header) (*types.Block, e
 	}
 
 	// set the timestamp
-	parentTime := time.Unix(int64(parent.Timestamp), 0)
-	headerTime := parentTime.Add(defaultBlockPeriod)
+	parentTime := time.UnixMilli(int64(parent.Timestamp))
```

Comment:
UnixMilli is added from go 1.17

Reply:
Resolved backward compatibility with two custom functions in consensus/utils.go that do the same thing 631515404cf370d1fc015938ea35da52ff9dbaaf

Reply:
You're linking the wrong commit but the change is there :) @ZeljkoBenovic  Resolving comment